Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gem 'opal', '0.9.0.beta2'
 #gem 'opal', :github => 'opal'
+gem 'rack', '1.6.4' # Opal compiler is *not* compatible with Rack 2.x
```

Comment:
opal depends on rack-test, which depends on rack >= 1.0. rack 2 was released June 30. So that explains it. (This is why I like depending on concrete versions. Other projects end up deciding when you upgrade.